features/ccx-upgrades-data-eng/caching.feature

```diff
@@ -0,0 +1,219 @@
+Feature: Upgrade Risks Prediction Data Engineering - test correct behavior of the LRU cache mechanism
+
+  # You have to add 127.0.0.1   localhost mock-oauth2-server
+  # to your /etc/hosts. This is needed to run these tests in Gitlab CI.
+
+  Background: Data eng service is running and well configured to work
+    Given The mock CCX Inference Service is running on port 9090
+    And The mock RHOBS Service is running on port 9091
+
+
+  Scenario: Check that nothing is cached if CACHE_ENABLED is set to 0
+    Given The CCX Data Engineering Service is running on port 8000 with envs
```

Comment:
I'd probably move it to background

Reply:
The env vars change so I can "setup" the cache on each test.